BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -130,6 +135,14 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
         brightnessChanged()
     }
 
+    public func brightnessChanged() {
+        if automaticBrightnessIsOn {
+            updateThemeBasedOnBrightness()
+        } else {
+            systemThemeChanged()
+        }
```

Comment:
I know this was part of the old code but you can actually remove this. If the brightness changes but the user doesn't have the automatic brightness change enabled the correct response is to do nothing.  No reason to set the theme at this point, it should be redundant.